lib/process-services-cloud/src/lib/people/components/people-cloud.component.ts

```diff
@@ -160,7 +161,8 @@ export class PeopleCloudComponent implements OnInit, OnChanges, OnDestroy {
     searchLoading = false;
 
     constructor(
-        private identityUserService: IdentityUserService,
+        @Inject(IDENTITY_USER_TOKEN)
```

Comment:
I guess you can do the same in all the places where we inject the identityUserService. Nothing will change because you will provide IdentityUserService by default but if we need to overwrite it we don't have to do any additional change in ADF

Reply:
As Maurizio suggested, we should do it 1 at time, so we can replace it in other places in the future.